cohortextractor/dsl.py

```diff
@@ -0,0 +1,185 @@
+"""This module provides classes for building a cohort using the DSL.
+
+Data lives in database tables, which are represented by PatientTable (a table with one
+row per patient) and EventTable (a table with multiple rows per patient).
+
+Through filtering, sorting, aggregating, and selecting columns, we transform instances
+of PatientTable/EventTable into instances of PatientSeries.
+
+A PatientSeries represents a mapping between patients and values, and can be assigned to
```

Comment:
> a mapping between patients and values

Maybe we can make this clearer? It's a one-to-one mapping, isn't it? And it's to a _single_ value (not a tuple of values like `*Frame`).

Reply:
(Both of those are implicit elsewhere, but I think it's worth being _super_ clear.)

Reply:
Will clarify.

Reply:
Partially addressed in 11ef1d41afdc547b0269232fcc39251c7f6516f0.  I think that this will become clearer with some good documentation.